src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/PartitionedExecution/FhirPartitionedExecutor.cs

```diff
@@ -110,28 +112,40 @@ private async Task<IEnumerable<TResult>> AnonymizeAsync(List<TSource> batchData,
                         throw new OperationCanceledException();
                     }
 
-                    try
+                    TResult anonymizedResult = await AnonymizerFunctionAsync(content);
+                    if (IsEmptyElementResult(anonymizedResult))
```

Comment:
`IsResultEmptyElement` looks a better name to me (just my personal perspective).

Reply:
Removed this function now.